js/adapt-offlineStorage-scorm.js

```diff
@@ -38,6 +38,7 @@ define([
       if (this.useTemporaryStore()) return temporaryStore[name];
 
       //Get by name
+      let courseState;
```

Comment:
Unsure why `courseState` is scoped to the function instead of `const`s inside the cases.

Reply:
Babel complains that it's a re-declaration.